src/docfx/config/AppData.cs

```diff
@@ -46,6 +49,11 @@ public static string GetCommitCachePath(string remote)
             return Path.Combine(CacheRoot, "commits", HashUtility.GetMd5Hash(remote));
         }
 
+        public static string GetCommitBuildTimePath(string remote, string branch)
+        {
+            return Path.Combine(CacheRoot, "history", $"build_history_{HashUtility.GetMd5Guid(remote)}_{HashUtility.GetMd5Guid(branch)}.json");
```

Comment:
>GetMd5Guid [](start = 116, length = 10)

The method is different with original one. Make sure the value will be same

Reply:
It is the same, I've updated test case to use `edge-developer` and the cache file path matches production path.